bip-XXXX.mediawiki

```diff
@@ -0,0 +1,138 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: Signatures of Messages using Bitcoin Private Keys
+  Author: Christopher Gilliard &ltchristopher.gilliard@gmail.com&gt
+  Comments-Summary: No comments yet.
+  Comments-URI: TBD
+  Status: Proposal
+  Type: Informational
+  Created: 2019-02-16
+  License: BSD-2-Clause
+</pre>
+
+# Abstract #
+
+This document describes a signature format for signing messages with Bitcoin private keys.
+
+The specification is intended to describe the standard for signatures of messages that can be signed and verfied between different clients that exist in the field today. Note: that a new signature format has been definied which has a number of advantages over this BIP, but to be backwards compatible with existing implementations this BIP will be useful. See BIP 322 [1] for full details on the new signature scheme.
```

Comment:
Typo: `definied` -> `defined`

Reply:
Fixed by this: https://github.com/bitcoin/bips/pull/764/commits/fe6899a294c5d437f727981f69a660a833bcba6b